README.md

```diff
@@ -77,6 +79,5 @@ x                    | @svenreimers@mastodon.social        | Sven Reimers
 @SvenRuppert         | @svenruppert@mastodon.social        | Sven Ruppert
 @synecdotal          | @synec@mastodon.social              | John Spurling
 @TonyPrintezis       | @TonyPrintezis@mastodon.social      | Tony Printezis
-@tnurkiewicz         | @nurkiewicz@mastodon.social         | Tomasz Nurkiewicz
```

Comment:
Are you sure you want to delete tnurkiewicz again?

Reply:
crap, no, that was an accident. just a sec...